tests/scan_network/check_vpn_firewall_test.py

```diff
@@ -0,0 +1,65 @@
+# stdlib
+import subprocess
+
+
+def docker_network_connect(cmd: str = "connect") -> None:
+    # this connects all the tailscale containers to the other docker compose project
```

Comment:
this test connects the containers to each others networks so that tailscale can discover a p2p direct route and speed up the netcat scan